README.md

```diff
@@ -32,6 +32,40 @@ For the moment, to build this project you should
 
 Service available on localhost:8080
 
+
+## Code style
+
+This project uses a consistent code style and provides definitions for use in both IntelliJ and Eclipse IDEs.
+
+For IntelliJ the process for importing the code style is:
+
+* Select *Settings* in the *File* menu
+* Select *Editor*
+* Select *Code Style*
+* Expand the menu item and select *Java*
+* Go to *Scheme* at the top, click on the setting button by the side of the drop-down list
+* Select *Import Scheme*
+* Select *IntelliJ IDE code style XML*
+* Navigate to the location of the file which relative to the project root is:  `.ide-settings/idea/OpenHospital-code-style-configuration.xml`
+* Select *OK* 
+* At this point the code style is stored as part of the IDE and is used for **all** projects opened in the editor.  To restrict the settings to just this project again select the setting button by the side of the *Scheme* list and select *Copy to Project...*. If successful a notice appears in the window that reads: *For current project*.
```

Comment:
I would suggest a collapsable section for the list, like this:

<details>
  <summary>IntelliJ instructions</summary>

* list1
* list2

</details>